pyquil/tests/test_operator_estimation.py

```diff
@@ -782,8 +848,8 @@ def test_exhaustive_symmetrization_2q(forest):
     assert np.isclose(frac7_0, expected_frac7_0, 2e-2)
 
 
-# TODO: gh-897
-@pytest.mark.skip
+# # TODO: gh-897
+# @pytest.mark.skip
```

Comment:
Why not just delete this rather than comment out?

Reply:
Done.